javascript/ql/src/semmle/javascript/JSX.qll

```diff
@@ -6,6 +6,13 @@ import javascript
 
 /**
  * A JSX element or fragment.
+ *
+ * Examples:
+ *
+ * ```
+ * <a href={linkTarget()}>{linkText()}</a>
+ * <><h1>Title</h1>Some <b>text</b></>
```

Comment:
Could you add an example with a non-HTML JSX element?

Reply:
:+1: Done.